gensim/models/ldamodel.py

```diff
@@ -1027,14 +1037,16 @@ def do_mstep(self, rho, other, extra_pass=False):
         logger.debug("updating topics")
         # update self with the new blend; also keep track of how much did
         # the topics change through this update, to assess convergence
-        diff = np.log(self.expElogbeta)
+        previous_Elogbeta = self.state.get_Elogbeta()
```

Comment:
are you sure then this is equivalent?

Reply:
Yeah, sure. `self.expElogbeta` are changed only in `__init__` and `sync_state`. `sync_state` are called here and in some workers, of course, to sync with current state given from master process/node. Also the main `self.state` are changed here too. In e-step  temporary state are changed everywhere (in ldamodel and atmodel, and of course ldamulticore/distributed).